pkg/api/configsync/v1alpha1/getters.go

```diff
@@ -22,7 +22,7 @@ import (
 // GetPeriodSecs returns the sync period defaulting to 15 if empty.
 func GetPeriodSecs(g *Git) float64 {
```

Comment:
Looks like this is unused - do you think it can be removed?

Reply:
Sure. I was thinking removing unused function seems to belong to a separate CL, but it's okay since it is a small change.
There is another unused function, which is also removed in this PR.

Reply:
I'm okay with removing it as a separate change, just wanted to mention it since I noticed it while reviewing this